base/libgit2/types.jl

```diff
@@ -963,11 +963,11 @@ for (typ, owntyp, sup, cname) in [
                 return obj
             end
         end
-        if isa(owntyp, Expr) && owntyp.args[1] == :Nullable
+        if isa(owntyp, Expr) && owntyp.args[1] == :Union && owntyp.args[2].args[1] == :Some
             @eval begin
-                $typ(ptr::Ptr{Void}, fin::Bool=true) = $typ($owntyp(), ptr, fin)
-                $typ(owner::$(owntyp.args[2]), ptr::Ptr{Void}, fin::Bool=true) =
-                    $typ($owntyp(owner), ptr, fin)
+                $typ(ptr::Ptr{Void}, fin::Bool=true) = $typ(nothing, ptr, fin)
+                $typ(owner::$(owntyp.args[2].args[2]), ptr::Ptr{Void}, fin::Bool=true) =
+                    $typ(Some(owner), ptr, fin)
```

Comment:
The Some wrapper seems unnecessary. That'll also remove the need for the second method here too.